.github/workflows/build_pkg.yml

```diff
@@ -34,20 +34,11 @@ jobs:
         with:
           fetch-depth: 0 # Fetch all history for all tags and branches
 
-      - name: Cache conda
-        uses: actions/cache@v3
-        env:
-          # Increase this value to reset cache if ./devtools/conda.recipe/build_env.yml has not changed
-          CACHE_NUMBER: 2
-        with:
-          path: ~/micromamba/pkgs
-          key: ${{ runner.os }}-conda-${{ env.CACHE_NUMBER }}-${{ hashFiles('./devtools/conda.recipe/build_env.yml') }}
-
       - name: Setup Conda Environment
-        uses: mamba-org/provision-with-micromamba@v11
+        uses: mamba-org/provision-with-micromamba@main
         with:
           environment-file: ./devtools/conda.recipe/build_env.yml
-          micromamba-version: 0.16.0
+          cache-downloads: true
```

Comment:
not using `cache-env` here because the whole build environment and artifacts end up in the environment bloating it up to >1GB

Reply:
Caching the archive downloads makes total sense. So the cache size is limited to 1GB currently?

Reply:
no I don't think there is a limit on the cache size but retrieving the huge environment cache was slower than just creating the build environment